src/libmeasurement_kit/common/utils.hpp

```diff
@@ -103,7 +104,18 @@ void timeval_now(timeval *tv) {
 
 double time_now();
 
+#ifdef _WIN32
+// Replacement implemented in utils.cpp
+tm *gmtime_r(const time_t *timep, tm *result);
+#endif
+
+// XXX unclear why this fails on Windows (yeah, now I really repent that
+// I did not implement a single mocking interface).
+#ifndef _WIN32
```

Comment:
TODO(@bassosimone): actually I later find out why in a Gist mentioned in this PR.